src/main/java/com/blamejared/compat/tcomplement/highoven/HighOven.java

```diff
@@ -0,0 +1,284 @@
+		public String describe() {
+			return String.format("Adding %s Recipe(s) for %s", this.name, this.getRecipeInfo());
+		}
+
+		@Override
+		protected String getRecipeInfo() {
+			return LogHelper.getStackDescription(output);
+		}
+	}
+
+	private static class RemoveHeat extends BaseAction {
+		private ILiquidStack input;
+		private ILiquidStack output;
+
+		public RemoveHeat(ILiquidStack input, ILiquidStack output) {
+			super("HighOven.HeatRecipe");
+			this.input = input;
+			this.output = output;
+		}
+
+		@Override
+		public void apply() {
+			REMOVED_HEAT_RECIPES.put(output, input);
+		}
+
+		@Override
+		public String describe() {
+			return String.format("Removing %s Recipe(s) for %s", this.name, this.getRecipeInfo());
+		}
+
+		@Override
+		public String getRecipeInfo() {
+			return LogHelper.getStackDescription(output);
+		}
+
+	}
+
+	@ZenMethod
+	public static void removeMixRecipe(ILiquidStack output) {
+		init();
+		CraftTweakerAPI.apply(new RemoveMix(output));
+	}
+
+	@ZenMethod
+	public static MixRecipeHelper newMixRecipe(ILiquidStack output, ILiquidStack input, int temp) {
+		init();
+		return new MixRecipeHelper(output, input, temp);
+	}
+
+	private static class RemoveMix extends BaseAction {
+		private ILiquidStack output;
+
+		public RemoveMix(ILiquidStack output) {
+			super("HighOven.MixRecipe");
+			this.output = output;
+		}
+
+		@Override
+		public void apply() {
+			REMOVED_MIX_RECIPES.add(output);
+		}
+
+		@Override
+		public String describe() {
+			return String.format("Removing %s Recipe(s) for %s", this.name, this.getRecipeInfo());
+		}
+
+		@Override
+		protected String getRecipeInfo() {
+			return LogHelper.getStackDescription(output);
+		}
+	}
+
+	@SubscribeEvent
+	public void onTinkerRegister(TCompRegisterEvent.HighOvenFuelRegisterEvent event) {
+		if (event.getRecipe() instanceof HighOvenFuelTweaker) {
+			return;
+		}
+		for (IItemStack entry : REMOVED_FUELS) {
+			if (event.getRecipe().matches(InputHelper.toStack(entry))) {
+				event.setCanceled(true);
+			}
+		}
+	}
+
+	@SubscribeEvent
+	public void onTinkerRegister(TCompRegisterEvent.HighOvenHeatRegisterEvent event) {
+		if (event.getRecipe() instanceof HeatRecipeTweaker) {
+			return;
+		} else {
+			for (Map.Entry<ILiquidStack, ILiquidStack> entry : REMOVED_HEAT_RECIPES.entrySet()) {
+				if (event.getRecipe().getOutput().isFluidEqual((FluidStack) entry.getKey().getInternal())) {
+					if (entry.getValue() != null) {
+						if (event.getRecipe().getInput().isFluidEqual((FluidStack) entry.getValue().getInternal())) {
+							event.setCanceled(true);
+						}
+					} else {
+						event.setCanceled(true);
+					}
+				}
+			}
+		}
+	}
+
+	@SubscribeEvent
+	public void onTinkerRegister(TCompRegisterEvent.HighOvenMixRegisterEvent event) {
+		if (event.getRecipe() instanceof MixRecipeTweaker) {
+			return;
+		} else {
+			for (ILiquidStack entry : REMOVED_MIX_RECIPES) {
+				if (event.getRecipe().getOutput().isFluidEqual((FluidStack) entry.getInternal())) {
```

Comment:
Diddo to above, use `event.getRecipe().matches(FluidStack,FluidStack)`. This also should probably check the input as an optional parameter for removing, in case there are multiple ways to get steel for example

Reply:
Added input to the check, otherwise same as above for check considerations.